wandb/apis/public/projects.py

```diff
@@ -148,3 +151,29 @@ def sweeps(self):
             )
             for e in ret["project"]["sweeps"]["edges"]
         ]
+
+    _PROJECT_ID = gql(
+        """
+        query ProjectID($projectName: String!, $entityName: String!) {
+            project(name: $projectName, entityName: $entityName) {
+                id
+            }
+        }
+        """
+    )
+
+    @property
+    def id(self) -> str:
```

Comment:
this is a change to the api, and since it is public method, it is probably good idea to update the change log